### PR TITLE
[Doc Improvement][Update application permissions]

### PR DESCRIPTION
**Pull request by v-abirade**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/428e0581eeff4a1c2d9231dbffc81e1e1421c018/msteams-platform/resources/schema/manifest-schema.md) | 91 | 33 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/12671ae0-1e2b-4437-8e4c-d8e1da2b09c9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6d6083dec417e046a58c8d1d75b55bc1a84a82c1/msteams-platform/resources/schema/manifest-schema.md) | 91 | 33 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ef4eeebd-5f17-492e-be39-d23b4ee99851) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/71b25d1ffb59f26dd6bfd4ca55f8d98059be643f/msteams-platform/resources/schema/manifest-schema.md) | 91 | 33 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e85b8108-2da7-4e80-9693-cdd2ec2b2812) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d48f6dc71089d33bdeba2b2adc590f79f57286fa/msteams-platform/resources/schema/manifest-schema.md) | 91 | 33 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/cffd750d-997d-46b1-88fe-4bcaeda4d0e5) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b1316b2d8c1992f10e7e1fde091322ac84c042d9/msteams-platform/resources/schema/manifest-schema.md) | 91 | 31 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/32c45996-eb0a-4c12-afbb-f3eeed371158) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/92380156fdbe19fb1bbaf035caad94ffbab1b697/msteams-platform/resources/schema/manifest-schema.md) | 91 | 31 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/61976849-c273-41a6-96e2-3bcc88b47b26) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6cf57dd8d1e4184f2e8ee573af50c4c45dd2d90b/msteams-platform/resources/schema/manifest-schema-dev-preview.md) | 84 | 52 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f41662dc-cb86-4127-825d-eebf27a08668) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6cf57dd8d1e4184f2e8ee573af50c4c45dd2d90b/msteams-platform/resources/schema/manifest-schema.md) | 91 | 31 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a95ae582-08e1-4b6f-a78e-746b5281b4ba) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/resources/schema/manifest-schema.md [hunk 1]
```diff
@@ -820,9 +820,13 @@ Specify and consolidate authorization related information for the app.
 |Name| Type|Maximum size|Required |Description|
 |---|---|---|---|---|
 |`type`|string||✔| The type of the resource-specific permission. Options: `Application` and `Delegated`.|
-|`name`|string|128 characters|✔|The name of the resource-specific permission. <br> For more information, see [Application permissions](../../graph-api/rsc/resource-specific-consent.md) and [Delegated permissions](#delegated-permissions).|
+|`name`|string|128 characters|✔|The name of the resource-specific permission. For more information, see [Application permissions](../../graph-api/rsc/resource-specific-consent.md) and [Delegated permissions](#delegated-permissions).|
 
-### Delegated permissions
+#### Application permissions
```
Comment (AkJo) [inline msteams-platform/resources/schema/manifest-schema.md:825]: We should perhaps call this

Resource-specific application permissions

File: msteams-platform/resources/schema/manifest-schema.md [hunk 2]
```diff
@@ -820,9 +820,13 @@ Specify and consolidate authorization related information for the app.
...
+
+For information on application permissions, see [RSC permissions for app manifest version 1.11 or earlier](../../graph-api/rsc/resource-specific-consent.md#manifest-changes-for-resource-specific-consent).
+
+#### Delegated permissions
```
Comment (AkJo) [inline msteams-platform/resources/schema/manifest-schema.md:829]: Resource-specific delegated permissions

File: msteams-platform/resources/schema/manifest-schema.md (hunk 1 above)
Comment (v-abirade) [reply msteams-platform/resources/schema/manifest-schema.md:825]: Thanks @AkJo , will update the same.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b1ac9638413922e7a6f7d961f0901885d3f64cc3/msteams-platform/resources/schema/manifest-schema-dev-preview.md) | 84 | 52 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/00397768-2f3a-4a8c-85c1-1cef3d80a1bc) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b1ac9638413922e7a6f7d961f0901885d3f64cc3/msteams-platform/resources/schema/manifest-schema.md) | 91 | 31 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/285c8fbc-2041-4e8b-be60-8ea188ce4e33) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/840ab178300a8e9d9382f2b24122cd0081317706/msteams-platform/resources/schema/manifest-schema-dev-preview.md) | 84 | 52 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ccf0ab94-a210-44f8-8835-ae8647b833db) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/840ab178300a8e9d9382f2b24122cd0081317706/msteams-platform/resources/schema/manifest-schema.md) | 91 | 31 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e081e37b-7711-4200-b604-dc7c031aaf39) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/resources/schema/manifest-schema-dev-preview.md [hunk 3]
```diff
@@ -649,9 +649,13 @@ Specify and consolidate authorization related information for the app.
 |Name| Type|Maximum size|Required |Description|
 |---|---|---|---|---|
 |`type`|string||✔| The type of the resource-specific permission. Options: `Application` and `Delegated`.|
-|`name`|string|128 characters|✔|The name of the resource-specific permission. <br> For more information, see [Application permissions](../../graph-api/rsc/resource-specific-consent.md) and [Delegated permissions](#delegated-permissions).|
+|`name`|string|128 characters|✔|The name of the resource-specific permission. For more information, see [Application permissions](../../graph-api/rsc/resource-specific-consent.md) and [Delegated permissions](#delegated-permissions).|
 
-### Delegated permissions
+#### Resource-specific application permissions
+
+For information on application permissions, see [RSC permissions for app manifest version 1.11 or earlier](../../graph-api/rsc/resource-specific-consent.md#manifest-changes-for-resource-specific-consent).
```
Comment (AkJo) [inline msteams-platform/resources/schema/manifest-schema-dev-preview.md:656]: Application permissions allow the app to access data without a signed-in user. For information on application permissions, see [Resource Specific Consent for MS Graph and MS BotSDK](../../graph-api/rsc/resource-specific-consent.md).

(Please use the right abbreviation as appropriate for MS and BotSDK).

File: msteams-platform/resources/schema/manifest-schema.md [hunk 4]
```diff
@@ -820,9 +820,13 @@ Specify and consolidate authorization related information for the app.
 |Name| Type|Maximum size|Required |Description|
 |---|---|---|---|---|
 |`type`|string||✔| The type of the resource-specific permission. Options: `Application` and `Delegated`.|
-|`name`|string|128 characters|✔|The name of the resource-specific permission. <br> For more information, see [Application permissions](../../graph-api/rsc/resource-specific-consent.md) and [Delegated permissions](#delegated-permissions).|
+|`name`|string|128 characters|✔|The name of the resource-specific permission. For more information, see [Application permissions](../../graph-api/rsc/resource-specific-consent.md) and [Delegated permissions](#delegated-permissions).|
 
-### Delegated permissions
+#### Resource-specific application permissions
+
+For information on application permissions, see [RSC permissions for app manifest version 1.11 or earlier](../../graph-api/rsc/resource-specific-consent.md#manifest-changes-for-resource-specific-consent).
```
Comment (AkJo) [inline msteams-platform/resources/schema/manifest-schema.md:827]: Exact same feedback here.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f6b4c62d22c1ac5fb332e181012c811cd8863cfa/msteams-platform/resources/schema/manifest-schema-dev-preview.md) | 84 | 52 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2b4bef68-a0ec-42ea-927e-e1dc2df246a5) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f6b4c62d22c1ac5fb332e181012c811cd8863cfa/msteams-platform/resources/schema/manifest-schema.md) | 91 | 31 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/051fe5ba-de17-43d7-8d5c-bce944bda634) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/054225bf5016aeb5919f6f349cd2bbcd0d3a4d2f/msteams-platform/resources/schema/manifest-schema-dev-preview.md) | 84 | 52 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f1a5bc41-5bb0-460c-9c9f-1af4c4b5cdb8) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/054225bf5016aeb5919f6f349cd2bbcd0d3a4d2f/msteams-platform/resources/schema/manifest-schema.md) | 91 | 31 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/408e7a73-ad7f-4d48-9482-4205df07cd1d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/resources/schema/manifest-schema.md (hunk 4 above)
Comment (v-abirade) [reply msteams-platform/resources/schema/manifest-schema.md]: Done

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e46d2ee46a34e180222aabefe5a82ff86d92c498/msteams-platform/resources/schema/manifest-schema-dev-preview.md) | 84 | 52 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7505f7d2-124d-4715-b44a-810c04b22cbe) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e46d2ee46a34e180222aabefe5a82ff86d92c498/msteams-platform/resources/schema/manifest-schema.md) | 91 | 31 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/23de4903-54b2-4c3c-9d2b-ba74b56e7f66) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/resources/schema/manifest-schema-dev-preview.md (hunk 3 above)
Comment (v-abirade) [reply msteams-platform/resources/schema/manifest-schema-dev-preview.md]: Updated.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/aab9caa2dfda94a897696dde8fae87de29cb2f8e/msteams-platform/resources/schema/manifest-schema-dev-preview.md) | 84 | 52 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/de6ecbd0-9ea4-4d62-811f-d26e08fbd073) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/aab9caa2dfda94a897696dde8fae87de29cb2f8e/msteams-platform/resources/schema/manifest-schema.md) | 91 | 31 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/27abb311-269f-49bf-b41a-8659abb4f0de) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9a02db44c19a79bab83305db6f01446e9fe3d360/msteams-platform/resources/schema/manifest-schema-dev-preview.md) | 84 | 52 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0112fe9e-5103-41b1-afa3-061892c97a50) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9a02db44c19a79bab83305db6f01446e9fe3d360/msteams-platform/resources/schema/manifest-schema.md) | 91 | 31 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c7b4a3ee-b977-47cc-8422-9a99aca23a3d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by AkJo — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]
Approved

**Pull request merged**